app/services/notifications/moderation.rb

```diff
@@ -4,6 +4,7 @@ module Moderation
     SUPPORTED = [Comment].freeze
 
     def self.available_moderators
+      # TODO: [@msarit] update query to call mod_roundrobin_notifications from correct table
```

Comment:
I need help reworking this query; `mod_roundrobin_notifications` now lives in `users_notification_setting`, not `users`.

Reply:
Same as above, re explicitly specifying where to find the column:

```ruby
User
  .with_role(:trusted)
  .joins(:notification_setting)
  .where("last_moderation_notification < ?", 3.days.ago)
  .where(notification_setting: { mod_roundrobin_notifications: true })
```